### PR TITLE
EDU-1061 Dashboard page improvements

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1061

### CODE REVIEW

File: src/components/user-card.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+profileButton:
+  en: view profile
```
Comment (ahelmberger) [inline src/components/user-card.yml:2]: Should be uppercased: "**V**iew profile" (like "Enter room", "Accept invitation", etc.)

**Comment by ahelmberger** [comment]
This is **so** good. The whole dashboard went from amateurish to starting-to-professional-looking, I cannot wait to see more improvements like these :heart_eyes::heart_eyes::heart_eyes:

File: src/components/user-card.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/user-card.yml]: Oh, corrected, thanks!

**Comment by adrianaluchian** [comment]
> This is **so** good. The whole dashboard went from amateurish to starting-to-professional-looking, I cannot wait to see more improvements like these 😍😍😍

Yes, it's amazing what an artistic eye can see!!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**